src/components/Widgets/SunInfo/index.tsx

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable react/function-component-definition */
+import { useSelector } from 'react-redux';
+import { weatherAllSelector } from 'store/weather-service/selectors';
+// types
+import { WeatherAll } from 'core/types';
+// img
+import sunRises from 'assets/img/sunrise.png';
+import sunSets from 'assets/img/sunset.png';
+// styles
+import s from './SunInfo.module.scss';
+
+export const SunInfo = () => {
+	const { daily }: WeatherAll = useSelector(weatherAllSelector);
+
+	const sunRise: Array<number> = daily.map(sun => sun.sunrise);
+	const sunSet: Array<number> = daily.map(sun => sun.sunset);
+	const minutesSunRise: string = String(new Date(sunRise[0] * 1000).getMinutes()).padStart(2, '0');
+	const minutesSunSet: string = String(new Date(sunSet[0] * 1000).getMinutes()).padStart(2, '0');
+
+	return (
+		<div className={s.suninfo}>
+			<h3>Sunrise & Sunset</h3>
+			<p>
+				<img src={sunRises} alt="" />
+				{new Date(sunRise[0] * 1000).getHours()}:{minutesSunRise}
```

Comment:
better will create a function to getting date or time

Reply:
done